docker/metrictank/metrictank.ini

```diff
@@ -170,3 +160,18 @@ consumer-opt =
 # max number of messages to allow in flight
 max-in-flight = 200
 
+
+[elasticsearch-idx]
+enabled = true
+# elasticsearch index name to use
+index = metric
+# Elasticsearch host addresses (multiple hosts can be specified as comma-separated list)
+hosts = elasticsearch:9200
+# how often the retry buffer should be flushed to ES
+retry-interval = 1h
```

Comment:
this is too long especially for short lived dev envs. let's stick to the default of 10m, or frankly even shorter